projects/ngx-datatable/src/lib/components/datatable.component.scss

```
@@ -216,7 +216,7 @@
     position: relative;
     z-index: 10;
     display: block;
-
+    overflow: hidden;
```

To remain consistent with the rest of the spacing and keep the semantics aligned:

```suggestion
    overflow: hidden;

```